waspc/examples/todoApp/ext/Main.css

```diff
@@ -1,64 +1,42 @@
-div {
-    color: green;
-}
-
-button.selected {
-    border: 1px solid black;
-}
-
-.hidden {
-    visibility: hidden;
-}
-
-.todos {
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-}
-
-.todos h1 {
-    text-align: center;
-}
-
-.todos__container {
-    width: 450px;
-}
-
-.todos__toggleAndInput {
-    display: flex;
-    align-items: flex-end;
-    margin-bottom: 10px;
-}
-
-.todos__toggleButton {
-    height: 40px;
-    margin-bottom: 8px;
-    margin-left: 20px;
-
-    background-color: white;
-    border: none;
-    display: inline-block;
-    font-size: 24px !important;
-    cursor: pointer;
-}
-
-.todos__newTaskForm {
-    width: 100%;
-    margin-left: 10px;
-}
-
-.todos__newTaskForm__input {
-  margin: 0 26px !important;
-}
-
-.todos__footer {
-    width: 100%;
-    margin-top: 10px;
-
-    display: flex;
-    justify-content: space-between;
-}
-
-.todos__footer__filters .filter:not(:last-child) {
-    margin-right: 10px;
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@layer base {
```

Comment:
this looks good to me! I also haven't really used @layer and @apply so far but from what I understood from the docs this is the right way to use it, to provide tailwind classes to some of your custom classes or html elements like you did here.